deepspeed/inference/engine.py

```diff
@@ -220,6 +220,10 @@ def build_alibi_tensor(self):
             if hasattr(self.module.transformer, 'build_mpt_alibi_tensor'):
                 self.module.transformer.build_mpt_alibi_tensor_orig = self.module.transformer.build_mpt_alibi_tensor
                 self.module.transformer.__class__.build_mpt_alibi_tensor = build_mpt_alibi_tensor
+        if hasattr(self.module, 'model'):
```

Comment:
@baodii - would it be possible to add a unit test for this model support?

Reply:
> @baodii - would it be possible to add a unit test for this model support?

@delock is it necessary to add a single model's autoTP unit test?

Reply:
I think we should add autoTP tests with a seperate workflow in a seperate PR, there are several reasons:
1. Current autotp coverage in inference UT is not adequate (marian, codegen only), many popular model are not tested
2. Popular model has large disk/memory requirement, put them in a seperate workflow allows us to isolate the hardware requirements
3. It is suggested to test these autotp test with `deepspeed` launcher rather than `pytest` distributed launcher, this could avoid additional complexity and as close to user environment as possible.

I think we can open an issue for this task and explore in a seperate task.